deploy/docker/cp-api-preconfigure/preconfigure-api

```diff
@@ -71,18 +103,46 @@ EOF
     rm -rf ${_cloud_data_name}-win32-x64
 }
 
+function configure_cloud_data_win_distribution() {
+    TEMP_PROCESSING_FILE="$CP_COMMON/win_processing.txt"
```

Comment:
As I can see now, `$CP_COMMON` by default points to `/opt/api/cp-common`. 
It is a container level directory now, but to synchronize multiple pods' startup in HA mode, `TEMP_PROCESSIG_FILE` should be stored in some shared persistent (host-based) location.

Please, modify deployment file, for `$CP_COMMON` to point onto some host volume.
And maybe:
- some more specific name, like `CP_CLOUD_DATA_PATCHING_DIR`, might be used, because `CP_COMMON` looks similar to `CP_API_COMMON_DIR`
- subdir in `CP_PREF_API_ETC_DIRECTORY` for those purpose could be created